src/voxel/world/meshing.rs

```diff
@@ -39,7 +39,7 @@ static SHARED_MESH_BUFFERS: Lazy<ThreadLocal<RefCell<MeshBuffers<Voxel, ChunkSha
     Lazy::new(ThreadLocal::default);
 
 /// Queues meshing tasks for the chunks in need of a remesh.
-fn queue_mesh_tasks(
+pub fn queue_mesh_tasks(
```

Comment:
Probably not right to keep these as pub. Instead we should have more strict ordering within the different `Sets`

Reply:
Should we centralize the comments on the main project. Will paste there